node/declaration.ts

```diff
@@ -1,10 +1,13 @@
-interface DeclarationWithPath {
+interface BaseDeclaration {
   function: string
+  displayName?: string
```

Comment:
This could actually be just `name`, since we're not using that anywhere, right?

Reply:
tbf I think the best would be to have

```ts
{
  name: string
  displayName: string
  ...
}
```

not sure why it is called `function` in the first place. But I think even if we do not change the `function` field I think `displayName` more clearly describes what it is, simply a name that is displayed in the UI.
Having `function` and `name` is confusing to me imho

Reply:
Scratch what I said above. I realized that this object actually represents one route, so `function` does make sense. And the `displayName` would be the name of the route kinda? So I guess I will change `displayName` to `name`.

Reply:
Yeah, exactly the `function` name makes sense because it's linking a path to a function.

Worth noting that in the future we'll be leaning more into in-source configuration, so `function` is not something that people will have to deal with. And I imagine that a `name` property there will make more sense to people than `displayName`, because it's not clear where that information will even be displayed on.